packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -117,21 +126,32 @@ const Summary: React.FunctionComponent< Props > = ( { siteId } ) => {
 							<Icon icon={ check } /> Builds brand recognition and trust
 						</li>
 					</ul>
-				</div>
-			</div>
-			<div className="focused-launch-summary__step">
-				<div className="focused-launch-summary__data-input">
-					<div className="focused-launch-summary__section">
-						<label className="focused-launch-summary__label">
-							{ __( '3. Confirm your plan', __i18n_text_domain__ ) }
-						</label>
-						<p className="focused-launch-summary__mobile-commentary focused-launch-summary__mobile-only">
-							<Icon icon={ bulb } /> Monetize your site with <strong>WordPress Premium</strong>
-						</p>
-						<Link to={ Route.PlanDetails }>{ __( 'View all plans', __i18n_text_domain__ ) }</Link>
-					</div>
-				</div>
-				<div className="focused-launch-summary__side-commentary">
+				</>
+			}
+		/>
+	);
+};
+
+interface PlanStepProps {
+	stepIndex: number;
+}
+
+const PlanStep: React.FunctionComponent< PlanStepProps > = ( { stepIndex } ) => {
+	return (
+		<SummaryStep
+			input={
+				<>
+					<label className="focused-launch-summary__label">
+						{ stepIndex }. { __( 'Confirm your plan', __i18n_text_domain__ ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( '3. Confirm your plan' )` *ES Score: 9*